.github/workflows/build-deb-package.yml

```diff
@@ -0,0 +1,52 @@
+name: Build Deb Package
+
+on:
+  push:
+    tags:
+    - '*'
```

Comment:
This is too general. This should be the tag of `jans-linux-installer`.

Reply:
@moabu could you post here jans-lixux-installer tag example